docs/troubleshooting.md

```diff
@@ -0,0 +1,25 @@
+# Troubleshooting
+
+This is a place to record workarounds to some common problems.
+
+## User didn't receive password reset email
+
+Signon current uses Amazon Simple Email Service (SES) to send emails to users. Amazon SES emails don’t pass strict DMARC setups. This means that some mail servers block signon email because they can't tell apart from phishing or spoof emails.
```

Comment:
```suggestion
Signon currently uses Amazon Simple Email Service (SES) to send emails to users. Amazon SES emails don’t pass strict DMARC setups. This means that some mail servers block Signon emails because they can't tell them apart from phishing or spoof emails.
```